common/persistence/cassandra/cluster_metadata_store.go

```diff
@@ -130,6 +137,14 @@ func (m *ClusterMetadataStore) SaveClusterMetadata(request *p.InternalSaveCluste
 	return true, nil
 }
 
+func (m *ClusterMetadataStore) DeleteClusterMetadata(request *p.InternalDeleteClusterMetadataRequest) error {
+	query := m.session.Query(templateDeleteClusterMetadata, constMetadataPartition, request.ClusterName)
```

Comment:
what would happen if current cluster is deleted from db?
should we prevent deleting current cluster?

Reply:
Delete current cluster is not allowed. This handles in the clusterMetadataStore.go in L165